Update testing for release

dustinswales opened the pull request:
This PR contains changes/updates to the SCM RT CI for the CCPP V7 release. Includes the following:
- SCM RT suite list updated to supported V7 SDFs/NML/tracers configurations.
- Working CCPP v6 and v5 SDF/nml/tracer configurations are still run as part of the SCM RT suite, but listed as unsupported options (i.e. If/when they break, we don't fix em).
- Plotting script for SCM RT vs. Baseline comparison step. Plots of all SCM outputs for all supported cases and suites are created and uploaded to GitHub as an "artifact"

This PR contains changes from #467 and #474.

grantfirl commented on .github/workflows/ci_run_scm_rts.yml:
```diff
@@ -150,6 +150,7 @@ jobs:
         cd ${SCM_ROOT}
         ./contrib/get_all_static_data.sh
         ./contrib/get_thompson_tables.sh
+        ./contrib/get_aerosol_climo.sh
```
This is going to slow down CI a lot. Any way to stash this data for future use?

dustinswales replied:
I was worried that this was going to bring this whole thing crashing down, but to my surprise it is faster to download this data on the GitHub servers than on Hera.[ It only took ~2min.](https://github.com/NCAR/ccpp-scm/actions/runs/9195025937/job/25289863152?pr=475), but that's more than the ~15sec before.
There may be slick ways to stash this, I just don't know any.

grantfirl commented on test/cmp_rt2bl.py:
```diff
@@ -25,6 +29,163 @@ def parse_args():
+    SCM_BL = Dataset(file_BL)
+    SCM_RT = Dataset(file_RT)
+
+    plot_files = []
```
This is a huge value-add. Thanks, @dustinswales !

grantfirl replied:
Would this make sense to have it in its own module in case we want to use this outside of the RT context? E.g. generalize it a bit to just plot the output by itself (no comparison) using a GitHub label or starting the CI workflow manually?

dustinswales replied:
Good idea, I will see how much of this I can get in this afternoon.

dustinswales replied:
done

grantfirl commented on test/cmp_rt2bl.py:
```diff
@@ -43,19 +204,39 @@ def main():
+            # end if
+
+            # Create plots between RTs and baselines
+            plot_files = plot_results(file_rt,file_bl)
```
Can we move this to only create plots if results are different than baselines? Otherwise, it's kinda wasted effort.

dustinswales replied:
Good point. Will do

dustinswales replied:
done.

grantfirl reviewed (approved):
Wonderful as-is, but there are a few questions to ponder.

dustinswales commented:
@grantfirl I addressed your concerns. Now the script only creates plots when there are differences, and the comparison/plotting scripts are usable outside of the RT/CI context (I used them offline for faster debugging of the CI plots)
All good to merege!
